functions/controllers/application/index.js

```diff
@@ -0,0 +1,147 @@
+const { uploadFile } = require("../../utils/storage");
+const { queryDocument, setDocument } = require("../../utils/database");
+
+const application = express();
+application.disable("x-powered-by");
+application.use(helmet());
+application.use(express.json());
+
+const api_url = issuer + "api/v2/";
+
+const corsOptions = {
+  origin: corsConfig,
+  optionsSuccessStatus: 200,
+};
+
+application.use(cors(corsOptions));
+
+/* TODO: 
+  Update createAppObject and validateAppObject with schema
+  Update document id posted (if necessary)
+  Update getNewFileName to get unique filenames
+  Unit Test Functions
+*/
+application.post("/submit", jwtCheck, hasUpdateApp, async (req, res) => {
+  try {
+    const form = new formidable.IncomingForm();
+    return await form.parse(req, async (err, fields, files) => {
+      if (err) {
+        functions.logger.log("Form Error: " + err);
+        return res.status(500).send({ code: 500, message: "Server Error" });
+      }
+      try {
+        // TODO: Update createAppObject and validateAppObject functions
+        appData = createAppObject(fields);
+        const isValidData = validateAppData(appData);
+        if (isValidData.length > 0) {
+          functions.logger.log(req.user.sub + " Validation Errors " + isValidData);
+          return res.status(400).send({ code: 400, message: "Form Validation Failed", errors: isValidData });
+        }
+        const isValidResume = validateResume(files);
+        if (isValidResume.length > 0) {
+          functions.logger.log(req.user.sub + " Validation Errors " + isValidResume);
+          return res.status(400).send({ code: 400, message: "Resume Validation Failed", errors: isValidResume });
+        }
+        // If bandwidth is too high, remove Auth0id validation
+        const token = await getM2MToken();
+        if (token === "") {
+          functions.logger.log("Failed to retrieve Token");
+          return res.status(500).send({ code: 500, message: "Server Error" });
+        }
+        const userInfoOptions = {
+          method: "GET",
+          url: api_url + "users/" + req.user.sub,
+          headers: { authorization: "Bearer " + token },
+        };
+
+        const userInfo = await axios(userInfoOptions);
+        if (!userInfo.data.email_verified) {
+          functions.logger.log("Unauthorized User: " + userInfo.data.email);
+          return res.status(403).send({ code: 403, message: "Unauthorized User" });
+        }
+        if (userInfo.data.email !== appData.email) {
+          functions.logger.log("Email/Authid Mismatch: " + userInfo.data.email + " " + appData.email);
+          return res.status(400).send({ code: 400, message: "Auth0Id discrepancy with Email" });
+        }
+
+        // Upload Resume Here
+        if (files && files.file) {
+          return (
+            // TODO: update getNewFileName() and replace test.pdf
+            uploadFile(storage, "resume", "test.pdf", files.file)
```

Comment:
Not sure if you still wanted these here considering the comment

Reply:
These will be updated once we decide on the schema and how we want to store our resumes based on formData.